packages/amplify-graphql-schema-generator/src/schema-generator/generate-schema.ts

```diff
@@ -370,6 +380,14 @@ export const convertToGraphQLTypeName = (modelName: string): string => {
   return singular(toPascalCase(cleanedInput?.split('_')));
 };
 
+export const convertToGraphQLFieldName = (fieldName: string): string => {
```

Comment:
What happens if the incoming name begins with a number?

Reply:
Updated to trim non-alphabetic characters from beginning of the field name.

Reply:
Please see new comment about leading underscore